src/lib/rest/rest.cpp

```diff
@@ -193,6 +195,23 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
       ciP->uriParamTypes.push_back(val);
     }
   }
+  else if (key == URI_PARAM_Q)
```

Comment:
Not sure of understanding this piece of code...

As far as I undertand, q filter is procesed and stored in a thread var stringFilterP. However, not sure were that variable is used after this parsing step...